packer/linux/conf/buildkite-agent/hooks/environment

```diff
@@ -10,8 +10,11 @@ BUILDKITE_DOCKER_CONFIG_TEMP_DIRECTORY=$(mktemp -d)
 export BUILDKITE_DOCKER_CONFIG_TEMP_DIRECTORY
 export DOCKER_CONFIG="$BUILDKITE_DOCKER_CONFIG_TEMP_DIRECTORY"
 
-if [ "${BUILDKITE_DOCKER_EXPERIMENTAL:-false}" = "true" ]
-then
+if [ "${BUILDKITE_DOCKER_EXPERIMENTAL:-false}" = "true" ]; then
+  if [ ! -f "${DOCKER_CONFIG}/config.json" ]; then
```

Comment:
I prefer `[[` over `[` because it's less bonkers, but I see `[` was already used here so 🤷‍♂️